package.json

```diff
@@ -45,6 +47,20 @@
     "automock": false,
     "setupFiles": [
       "<rootDir>/setupJest.js"
-    ]
+    ],
+    "coverageThreshold": {
+      "global": {
+        "branches": 100,
+        "functions": 100,
+        "lines": 100,
+        "statements": 100
+      }
+    }
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run lint && npm run test",
+      "pre-push": "npm run lint && npm run test"
+    }
```

Comment:
@davidcostadev , in my opinion, it doesn't make much sense to have the pre-commit and the pre-push hooked to the same thing.

I would suggest one of three things:
- removing the pre-push hook, since lint and test are ran before commit (and commit always precedes push);
- removing the pre-commit hook and letting the developer deal with code and test inconsistencies only before pushing;
- let the pre-commit hook deal only with linting, while the pre-push does both.

I prefer the first option, but it's up to you =)

Reply:
I've fixed that and merged it into master ;]